### ISSUE TITLE
Ranch version requirement catch-22

### ISSUE THREAD

**Issue by arjan** [gen-smtp/gen_smtp#242]
I have an issue with publishing the package now... :-| 

I cannot publish a hex version requirement `>= 1.7` (see below)
however when I use a spec like `~> 1.7 or ~> 2.0` (which works with elixir), it will not work with Erlang.

it seems like a real catch-22...

```
===> Failed to publish package: Validation error(s)
	Requirements could not be computed
ranch
--------------------
invalid requirement: ">= 1.7"
```

@mworrell @seriyps If any of you know a solution..

**Comment by arjan** [comment]
The only thing I can think of is dropping ranch < 2.0...

**Comment by arjan** [comment]
Nevermind, fixed! It should have been `>= 1.7.0`, eg a full semver.
